controllers/pipelines/internal/workflowutil/workflow_utils_test.go

```diff
@@ -1,14 +1,20 @@
 //go:build unit
 
-package pipelines
+package workflowutil
 
 import (
 	argo "github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"testing"
 )
 
+func TestWorkflowUtilsUnitSuite(t *testing.T) {
```

Comment:
Just an FYI in case anyone wants to do similar refactorings in the future: ginko hooks into a suite function, but since this test used to rely on the `suite_unit_test.go` on the `controllers/pipelines` package, it won't actually run this test and you won't know unless you look at what tests were ran. There are a few ways around this, e.g. register tests explicitly (see ginko docs). But I decided to make a new suite, which according to the docs is not bad practice either.

This is also why `workflowfactory` package has it's own unit and integration suites.